unwrap-smt/README.md

```diff
@@ -0,0 +1,123 @@
+
+```
+                   +-------------+
+                   |             |
+                   |    MySQL    |
+                   |             |
+                   +------+------+
+                          |
+                          |
+                          |
+          +---------------v------------------+
+          |                                  |
+          |           Kafka Connect          |
+          |  (Debezium, JDBC, ElasticSearch  |
+          |           connectors)            |
+          |                                  |
+          +---+-----------+------------------+
+                          |
+                          |
+                          |
+                          |
+                  +-------v--------+
+                  |                |
+                  |   PostgreSQL   |
+                  |                |
+                  +----------------+
+
+
+```
+We are using a Docker Compose to deploy following components
+* MySQL
+* Kafka
+  * ZooKeeper
+  * Kafka Broker
+  * Kafka Connect with [Debezium](http://debezium.io/) and  [JDBC](https://github.com/confluentinc/kafka-connect-jdbc) Connectors
+* PostgreSQL
+
+## Usage
+How to run:
+
+```shell
+# Start the application
+DEBEZIUM_VERSION=0.6
+docker-compose up
+
+# Start PostgreSQL connector
+curl -i -X POST -H "Accept:application/json" -H  "Content-Type:application/json" http://localhost:8083/connectors/ -d @jdbc-sink.json
+
+# Start MySQL connector
+curl -i -X POST -H "Accept:application/json" -H  "Content-Type:application/json" http://localhost:8083/connectors/ -d @source.json
+```
+
+Check content of MySQL database
+```shell
+docker-compose exec mysql bash -c 'mysql -u $MYSQL_USER  -p$MYSQL_PASSWORD inventory -e "select * from customers"'
++------+------------+-----------+-----------------------+
+| id   | first_name | last_name | email                 |
++------+------------+-----------+-----------------------+
+| 1001 | Sally      | Thomas    | sally.thomas@acme.com |
+| 1002 | George     | Bailey    | gbailey@foobar.com    |
+| 1003 | Edward     | Walker    | ed@walker.com         |
+| 1004 | Anne       | Kretchmar | annek@noanswer.org    |
++------+------------+-----------+-----------------------+
+```
+
+Verify that PostgreSQL database has the same content
+```shell
+docker-compose exec postgres bash -c 'psql -U $POSTGRES_USER $POSTGRES_DB -c "select * from customers"'
+ last_name |  id  | first_name |         email         
+-----------+------+------------+-----------------------
+ Thomas    | 1001 | Sally      | sally.thomas@acme.com
+ Bailey    | 1002 | George     | gbailey@foobar.com
+ Walker    | 1003 | Edward     | ed@walker.com
+ Kretchmar | 1004 | Anne       | annek@noanswer.org
+(4 rows)
+```
+### New record
+Insert a new record into MySQL
+```shell
+docker-compose exec mysql bash -c 'mysql -u $MYSQL_USER  -p$MYSQL_PASSWORD inventory'
+mysql> insert into customers values(default, 'Reinhard', 'Scheer', 'cic@hsf.de');
```

Comment:
Who is this dude ;)

Reply:
Let's say that I've an unusual hobby ;-) https://de.wikipedia.org/wiki/Skagerrakschlacht